### PR TITLE
[flutter_release] Update dart_style to published version 2.0.2

### PR DESCRIPTION
**Pull request by CaseyHillers**

### CODE REVIEW

File: DEPS [hunk 1]
```diff
@@ -352,7 +352,7 @@ deps = {
    Var('dart_git') + '/yaml.git@b4c4411631bda556ce9a45af1ab0eecaf9f3ac53',
 
   'src/third_party/dart/third_party/pkg_tested/dart_style':
-   Var('dart_git') + '/dart_style.git@0fe592042eda5d9eea6fc240a8ee0fe531bb0794',
+   Var('dart_git') + '/dart_style.git@09d9dff90d9a2e0793ad2f795f36c2777f720eda0',
```
Comment (christopherfujino) [inline DEPS:355]: looks like another bad copy paste? you have an extra leading 0

**Review by christopherfujino — CHANGES_REQUESTED** [review]
`9d9dff90d9a2e0793ad2f795f36c2777f720eda0`

File: DEPS (hunk 1 above)
Comment (CaseyHillers) [reply DEPS]: Nice catch! Done

File: DEPS [hunk 2]
```diff
@@ -352,7 +352,7 @@ deps = {
    Var('dart_git') + '/yaml.git@b4c4411631bda556ce9a45af1ab0eecaf9f3ac53',
 
   'src/third_party/dart/third_party/pkg_tested/dart_style':
-   Var('dart_git') + '/dart_style.git@0fe592042eda5d9eea6fc240a8ee0fe531bb0794',
+   Var('dart_git') + '/dart_style.git@09d9dff90d9a2e0793ad2f795f36c2777f720eda',
```
Comment (christopherfujino) [inline DEPS:355]: I think you removed the wrong 0 :)

Comment (CaseyHillers) [reply DEPS]: You passed the code review test :)

**Review by christopherfujino — APPROVED** [review]
LGTM

File: DEPS (hunk 2 above)
Comment (christopherfujino) [reply DEPS]: i should, all reviewing these release PRs consists of is validating hashes :P

**Review by christopherfujino — CHANGES_REQUESTED** [review]
Actually, this also requires an update to the licenses golden.

In Linux unopt, we check the licenses, and it's failing:

```
-Signature: 3819f11e399597681e8b8b7e58e60404
+Signature: e4244c3d5f0a164de1f02f6c23f5e9ef
```

**Comment by christopherfujino** [comment]
Actually, in "Linux Unopt", the licenses check 

> Actually, this also requires an update to the licenses golden.
> 
> In Linux unopt, we check the licenses, and it's failing:
> 
> ```
> -Signature: 3819f11e399597681e8b8b7e58e60404
> +Signature: e4244c3d5f0a164de1f02f6c23f5e9ef
> ```

That diff needs to be applied to the first line of:

https://raw.githubusercontent.com/flutter/engine/flutter-2.4-candidate.0/ci/licenses_golden/licenses_third_party

**Comment by CaseyHillers** [comment]
> That diff needs to be applied to the first line of:
> 
> https://raw.githubusercontent.com/flutter/engine/flutter-2.4-candidate.4/ci/licenses_golden/licenses_third_party

Done

**Review by christopherfujino — APPROVED** [review]
LGTM. I'll wait for CI to finish till I merge.

**Comment by christopherfujino** [comment]
licenses_check looks good, I'll merge now.

**Pull request merged**